src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { Messaging } from './Messaging';
```

Comment:
is this sufficient as `Messaging` only exports it's class and not the interfaces, events and so on?